### PR TITLE
Card and about sections

### PR DESCRIPTION
**Pull request by Mikelobi**
### Pop up window modal

In this Project I: 

-  Added Popup modal to the portfolio card section.
- Checked for linters error.
- Updated readme markdown.

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Get the modal
+const modal = document.getElementById('myModal');
+const modal2 = document.getElementById('myModal2');
+const modal3 = document.getElementById('myModal3');
+const modal4 = document.getElementById('myModal4');
+const modal5 = document.getElementById('myModal5');
+const modal6 = document.getElementById('myModal6');
+const modal7 = document.getElementById('myModal7');
```
Comment (ReshyResh) [inline popup.js:8]: - [ ] Following the above comment, please keep only one modal, and change the informations based on project objects. As per requirement, you should create an array of objects containing data about each project, which you'll be using to fill the informations inside both the modal window and the project card itself. 
Example: 
```
const projects = [{
    name: "Project one",
    description: "Description one",
    image: "link_to_image.jpg",
    technologies: ["CSS", "JavaScript", "HTML"],
    link: "link_to_live_version.com",
}]

```

File: index.html [hunk 2]
```diff
@@ -217,7 +217,316 @@ <h2 class="card-heading">Multi-Post Stories</h2>
...
+                    </ul>      
+                  </div>
+                  <div class="modal-body">
+                    <img src="image/Snapshoot-Portfolio-4.png" alt="a baby" class="web">
+                    <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page" class="mob">
+                  </div>
+                  <div class="web-modal">
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent<span class="mode-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.</span></p>
+                    <div class="links-modal">
```
Comment (ReshyResh) [inline index.html:511]: - [ ] Please make sure this text matches the Figma template in terms of text color and font-family, font-size, for a better UI.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @Mikelobi , 

### Good job so far!
  - ✔ Modals are working.
  - ✔ Design follows the guidelines aside for some small errors.
  - ✔ No linter errors.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Please make sure to follow the requirements and create an array of objects which gets used both for generating the projects section and the modal popups.

_Please check the comments under the review for more comments._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popup.js (hunk 1 above)
Comment (Kingstalux) [reply popup.js]: resolved

File: index.html [hunk 3]
```diff
@@ -217,7 +217,316 @@ <h2 class="card-heading">Multi-Post Stories</h2>
                 <li><a href="#">css</a></li>
                 <li><a href="#">javascript</a></li>
               </ul>
-              <button type="button" class="button detail-style2">See Project</button>
+              <button id="myBtn7" type="button" class="button detail-style2">See Project</button>
+            </div>
+          </div>
```
[the remark this answers is not in the document]
Comment (Kingstalux) [reply index.html]: not resolved

File: style.css [hunk 4]
```diff
@@ -261,6 +261,159 @@ header {
...
+
+.modal-live li {
+  margin-right: 20px;
+}
+
+a.button.modal-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 160px;
```
[the remark this answers is not in the document]
Comment (Kingstalux) [reply style.css:387]: resolved  to a certain extend

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Mikelobi , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f644550476b6e5269456e6965632f67697068792e676966


 ## Required Changes ♻️
_Check the comments under the previous reviewer's comments._
- [ ] You are to delete all the hard-coded HTML in the project section and generate it dynamically using JavaScript when the page loads but I noticed that all your projects are still hardcoded in the HTML file. It would be great if you could implement this functionality for a better user interface and also to follow the requirements.
- Your project HTML section:
Annotation 2022-03-10 162037



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
## Hi *** @Mikelobi , *** 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**